### PR TITLE
Add and remove figures in Cubeviz plugins docs

### PR DESCRIPTION
**Pull request by PatrickOgle**
Added figures for Collapse, Moment Maps, and Line Maps sections of Cubeviz Plugins RTD.

### CODE REVIEW

**Comment by PatrickOgle** [comment]
@javerbukh How do I remove this src/specutils directory from this documentation PR?  Not sure where it came from.

**Comment by PatrickOgle** [comment]
Added links to Specviz in MOSviz Plugins documentation.

**Comment by PatrickOgle** [comment]
> @javerbukh How do I remove this src/specutils directory from this documentation PR? Not sure where it came from.

I was able to remove it from git tracking.

File: docs/cubeviz/plugins.rst [hunk 1]
```diff
@@ -65,23 +69,23 @@ also saved to a dataset with the label specified in the :guilabel:`Model Label`
...
+the :guilabel:`Linear` icon for auto-contours or the
+:guilabel:`Custom` icon to set your own levels.
 
-.. image:: img/contours_activated.png
 
 The :guilabel:`Contours` of a 2D image can also be plotted over an image viewer.
-A user can create a 2D image using a plugin like :ref:`Collapse <collapse>` and then add it to an image viewer as a 2nd layer.
+A user can create a 2D image using a plugin like :ref:`Collapse <collapse>` and then add it
+to an image viewer as a 2nd layer. To visualize the contours of this 2D image over the image viewer,
+first go to the :guilabel:`Layer` tab, select the 2nd layer (which should be the 2D image), and
```
Comment (javerbukh) [inline docs/cubeviz/plugins.rst:83]: First should be "add the data to the data drop down tab", then "go to the layer tab and select the 2nd layer." Although, the image already in the docs shows the second data being selected, so I will leave it up to your discretion whether or not to add this.

**Review by javerbukh — APPROVED** [review]
Left one comment but otherwise this looks great! The documentation is clear enough that I as a astronomy newbie understood everything, hopefully it makes sense the other way around as well! 😄

File: docs/cubeviz/plugins.rst (hunk 1 above)
Comment (PatrickOgle) [reply docs/cubeviz/plugins.rst]: I have made this change, but also rearranged the text a bit to say that contours of any second image can be overplotted, and that the Collapse image contours in the figure are one particular example of this.

**Review by rosteen — APPROVED** [review]
Everything looks good to me, will merge shortly.

**Pull request merged**